spec/pq/authentication_methods_spec.cr

```diff
@@ -9,38 +9,39 @@ require "../spec_helper"
 
 describe PQ::Connection, "nologin role" do
   it "raises" do
-    DB.exec("drop role if exists crystal_test")
-    DB.exec("create role crystal_test nologin")
+    PG_DB.exec("drop role if exists crystal_test")
+    PG_DB.exec("create role crystal_test nologin")
     expect_raises(PQ::PQError) {
-      PG::Connection.new("host=localhost user=crystal_test")
+      DB.open("postgres://crystal_test@localhost")
     }
-    DB.exec("drop role if exists crystal_test")
+    PG_DB.exec("drop role if exists crystal_test")
   end
 end
 
 if File.exists?(File.join(File.dirname(__FILE__), "../.run_auth_specs"))
   describe PQ::Connection, "md5 auth" do
     it "works when given the correct password" do
-      DB.exec("drop role if exists crystal_md5")
-      DB.exec("create role crystal_md5 login encrypted password 'pass'")
-      conn = PG::Connection.new("host=localhost user=crystal_md5 password=pass")
-      conn.exec("select 1").rows.first.first.should eq(1)
-      DB.exec("drop role if exists crystal_md5")
+      PG_DB.exec("drop role if exists crystal_md5")
+      PG_DB.exec("create role crystal_md5 login encrypted password 'pass'")
+      DB.open("postgres://crystal_md5:pass@localhost") do |db|
+        db.query_one("select 1", &.read).should eq(1)
+      end
```

Comment:
For the record, i think the github way to patch a PR-1 is send a PR-2 based on the branch the PR-1 is based on.

For the length of the patch your post is clearly lighter ;-)